### PR TITLE
Creating Lisbon city file.

### LINKED ISSUE #75: adrianorsimoes hometown file
create hometown file

### PR DESCRIPTION
**Pull request by adrianosimoes**
This PR resolves #75.

### CODE REVIEW

File: adrianorsimoes-lisbon.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Lisbon, Portugal
+
+
+### Place to Eat:
+ Tavares
+ Portugalia
+ 
+
+### Fun things to do:
```
Comment (adrianosimoes) [inline adrianorsimoes-lisbon.md:10]: @pbo-msp  What things can we do here?

Comment (pbo-moviestarplanet) [inline adrianorsimoes-lisbon.md:10]: @adrianosimoes How would I know, I haven't been to lisbon

**Comment by pbo-moviestarplanet** [comment]
:camel:

File: adrianorsimoes-lisbon.md (hunk 1 above)
Comment (adrianosimoes) [inline adrianorsimoes-lisbon.md:10]: The bridge? :)

**Comment by pbo-moviestarplanet** [comment]
:camel:

**Pull request merged**